README.adoc

```diff
@@ -195,6 +195,22 @@ You can also confirm the installation of the operators from the web console.
 Navigate to the OperatorHub.
 From the categories on the left, you can filter to see only installed operators. 
 
+== Adding a private Docker credential
+
+Docker limits container image pull requests for free DockerHub subscriptions. 
+Read the link:https://www.docker.com/increase-rate-limits[Understanding Docker Hub Rate Limiting] document for more details.
```

Comment:
suggested edit:

Read the link:https://www.docker.com/increase-rate-limits[Understanding Docker Hub Rate Limiting] document for more details.
If you have a Docker account with a Pro or Team subscription, you can add a private credential to avoid running into any errors as a result of the limits.
--->
For more information, see link:https://www.docker.com/increase-rate-limits[Understanding Docker Hub Rate Limiting].
If you have a Docker account with a Pro or Team subscription, you can add a private credential to avoid any errors as a result of the limits.